app/communication/views/banner.py

```diff
@@ -0,0 +1,38 @@
+from django.db.models import Q
+from rest_framework import status
+from rest_framework.decorators import action
+from rest_framework.response import Response
+
+from app.common.mixins import ActionMixin
+from app.common.pagination import BasePagination
+from app.common.permissions import BasicViewPermission, IsHS
+from app.common.viewsets import BaseViewSet
+from app.communication.models.banner import Banner
+from app.communication.serializers.banner import BannerSerializer
+from app.util.utils import now
+
+
+class BannerViewSet(BaseViewSet, ActionMixin):
+    serializer_class = BannerSerializer
+    pagination_class = BasePagination
+    queryset = Banner.objects.all()
+    permission_classes = [IsHS]
```

Comment:
Denne må endres til `permission_classes = [BasicViewPermission]` for at permissionsene i modellen skal brukes 😉